doc/contributor/howto-guide-set-up-development-environment.md

```diff
@@ -268,7 +270,9 @@ pip install -r ci/requirements.txt
 use with:
 
 ```bash
-git ls-files -z -- '*.md' ':!:**/testdata/**' | xargs -0 -r -P "$(nproc)" -n 50 mdformat
+git ls-files -z -- \
+    '*.md' ':!:**/testdata/**' ':!:**/generated/**' | \
+    xargs -0 -r -P "$(nproc)" -n 50 mdformat
```

Comment:
nit: I do not think we have enough `.md` files to justify the complex `xargs` arguments, maybe just do the same as with the other commands:

```suggestion
    xargs -0 mdformat
```

Reply:
Updated here and in `lint.yaml`